source-repo-scripts/merge_forward_pull_request.bash

```diff
@@ -58,4 +61,5 @@ gh pr create \
     --repo "$ORIGIN_ORG_REPO" \
     --base "$TO_BRANCH" \
     --body "$BODY" \
-    --head "$CURRENT_BRANCH"
+    --head "$CURRENT_BRANCH" \
+    --web
```

Comment:
this script doesn't work well for me anymore since these changes, using the latest `gh` from homebrew. It used to just open the pull request with the body completely filled out, but now it opens a web browser without filling in everything. @azeey which version of `gh` have you tested with?

Reply:
I have `2.27.0`.  I didn't actually mean to make this change in this PR, so we can remove it if it's not working for you. I do like the `--web` flag though, since I can review the PR body/title before creating it or avoid accidentally creating PRs.